customer/src/test/java/no/unit/nva/customer/NvaApplicationDomainHandlerTest.java

```diff
@@ -0,0 +1,98 @@
+package no.unit.nva.customer;
+
+import static no.unit.nva.customer.testing.CustomerDataGenerator.randomCristinOrgId;
+import static no.unit.nva.customer.testing.CustomerDataGenerator.randomPublicationWorkflow;
+import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValuesIgnoringFields;
+import static no.unit.nva.testutils.RandomDataGenerator.randomElement;
+import static no.unit.nva.testutils.RandomDataGenerator.randomInteger;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.URI;
+import java.time.Instant;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import no.unit.nva.customer.model.ApplicationDomain;
+import no.unit.nva.customer.model.CustomerDto;
+import no.unit.nva.customer.model.VocabularyDto;
+import no.unit.nva.customer.model.VocabularyStatus;
+import no.unit.nva.customer.service.impl.DynamoDBCustomerService;
+import no.unit.nva.customer.testing.LocalCustomerServiceDatabase;
+import no.unit.nva.stubs.FakeContext;
+import nva.commons.core.attempt.Try;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class NvaApplicationDomainHandlerTest extends LocalCustomerServiceDatabase {
+
+    private NvaApplicationDomainHandler handler;
+    private DynamoDBCustomerService service;
+    private Context context;
+    private Void input;
+
+    @BeforeEach
+    public void setUp() {
+        super.setupDatabase();
+        service = new DynamoDBCustomerService(this.dynamoClient);
+        handler = new NvaApplicationDomainHandler(service);
+        context = new FakeContext();
+    }
+
+    @Test
+    void shouldUpdateCustomerOfAttribute() {
+        var expectedCustomersAppDomains = IntStream.of(1, randomInteger(10))
```

Comment:
can randomInteger(10) yield 0? if so you might get a runtime error randomly. (creating an instream that starts on 1 and continures up to 0).

I think to ensure that this does not happen the IntStream.of(0, randomInteger(10)) this will prevent the runtime error, but you might end up with a list with 0 elements which might be okay.